guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -683,7 +679,7 @@ NOTE: Before working on older versions, please check the [maintenance policy](ma
 
 #### Backporting
 
-Changes that are merged into main are intended for the next major release of Rails. Sometimes, it might be beneficial for your changes to propagate back to the maintenance releases for older stable branches. Generally, security fixes and bug fixes are good candidates for a backport, while new features and patches that introduce a change in behavior will not be accepted. When in doubt, it is best to consult a Rails team member before backporting your changes to avoid wasted effort.
+Changes that are merged into main are intended for the next major release of Rails. Sometimes, it might be beneficial for your changes to propagate back to older stable branches' maintenance releases. Generally, security fixes and bug fixes are good candidates for a backport, while new features and patches that introduce a behavior change will not be accepted. When in doubt, it is best to consult a Rails team member before backporting your changes to avoid wasted effort.
```

Comment:
```suggestion
Changes that are merged into main are intended for the next major release of Rails. Sometimes, it might be beneficial to propagate your changes back to older stable branches for inclusion in maintenance releases. Generally, security fixes and bug fixes are good candidates for a backport, while new features and patches that change expected behavior will not be accepted. When in doubt, it is best to consult a Rails team member before backporting your changes to avoid wasted effort.
```